### PR TITLE
Docs(config) : Add cache.cacheLocation

### PR DESCRIPTION
**Pull request by jeffin143**
In reference with https://github.com/webpack/webpack/blob/d00e65529a4c5a4e91843f2d4178c66eff3784ae/schemas/WebpackOptions.json#L494

- [x] Read and sign the [CLA][1]. PRs that haven't signed it won't be accepted.
- [x] Make sure your PR complies with the [writer's guide][2].
- [x] Review the diff carefully as sometimes this can reveal issues.
- [x] Do not abandon your Pull Request: [Stale Pull Requests][3].

### CODE REVIEW

File: src/content/configuration/other-options.md [hunk 1]
```diff
@@ -126,6 +126,26 @@ module.exports = {
 
 W> The final location of the cache is a combination of `cache.cacheDirectory` + `cache.name`.
 
+### `cache.cacheLocation`
+
+`string`
+
+Locations for the cache. Defaults to `cache.cacheDirectory` / `cache.name`.
```
Comment (montogeek) [inline src/content/configuration/other-options.md:133]:
```suggestion
Locations for the cache. Defaults to `cache.cacheDirectory` or `cache.name`.
```

Comment (jeffin143) [reply src/content/configuration/other-options.md]: IG it is `cache.cacheDirectory/cache.name`

Comment (montogeek) [reply src/content/configuration/other-options.md]: Maybe it is better to writing it like JS template literals?`${cache.cacheDirectory}/${cache.name}`

Not sure.

Comment (jeffin143) [reply src/content/configuration/other-options.md]: +1 for that :)

Comment (EugeneHlushko) [reply src/content/configuration/other-options.md]: Yeah Fernando's suggestion looks good although we could even show the default including path call `path.resolve(cache.cacheDirectory, cache.name)` to be on the safe side with odd paths given WDYT?

File: src/content/configuration/other-options.md [hunk 2]
```diff
@@ -126,6 +126,26 @@ module.exports = {
 
 W> The final location of the cache is a combination of `cache.cacheDirectory` + `cache.name`.
 
+### `cache.cacheLocation`
+
+`string`
+
+Locations for the cache. Defaults to `cache.cacheDirectory/cache.name`.
```
Comment (EugeneHlushko) [inline src/content/configuration/other-options.md:133]: I still think this is false
```
path.resolve(cache.cacheDirectory, cache.name) !== `${cache.cacheDirectory}/${cache.name}`
```

Comment (jeffin143) [reply src/content/configuration/other-options.md:133]: I guess you are right, let's go with path.resolve(cache.cacheDirectory, cache.name)

**Comment by jeffin143** [comment]
IG This is done, Lets go ahead with this

**Comment by EugeneHlushko** [comment]
Thanks!

**Pull request merged**